app/views/matches/_match_history.html.erb

```diff
@@ -32,14 +33,17 @@
         <% if show_thrower_leaver_column %>
           <th class="match-header hide-sm tooltipped tooltipped-n" aria-label="Throwers and leavers">😢</th>
         <% end %>
+        <% if show_potg_column %>
+          <th class="match-header hide-sm tooltipped tooltipped-n" aria-label="Play of the Game">🏆</th>
+        <% end %>
         <% if can_edit %>
           <th class="match-header"><span class="ion-gear-b ion"></span></th>
         <% end %>
       </tr>
     </thead>
     <tbody class="js-filterable-matches">
       <% matches.each_with_index do |match, i| %>
-        <tr id="match-row-<%= match.id %>" class="js-filterable-match js-thrower-leaver-<%= (match.thrower? || match.leaver?) ? 'y' : 'n' %> js-time-<%= match.day_of_week %>-<%= match.time_of_day %> js-group-size-<%= match.group_size %> js-result-<%= match.result %> <%= 'match-placement-log-row' if match.placement_log? %> <%= 'match-placement-row' if match.placement? %> <%= 'match-last-placement-row' if match.placement? && i == Match::TOTAL_PLACEMENT_MATCHES - 1 %>">
+        <tr id="match-row-<%= match.id %>" class="js-filterable-match js-thrower-leaver-<%= (match.thrower? || match.leaver?) ? 'y' : 'n' %> js-time-<%= match.day_of_week %>-<%= match.time_of_day %> js-group-size-<%= match.group_size %> js-result-<%= match.result %> <%= 'match-placement-log-row' if match.placement_log? %> <%= 'match-placement-row' if match.placement? %> <%= 'match-last-placement-row' if match.placement? && i == Match::TOTAL_PLACEMENT_MATCHES - 1 %> js-potg-<%= match.potg_earned? ? 'y' : 'n' %>">
```

Comment:
Could modify the potg_char method to return “n” as well, then use that method here for the class. Or, since you only care about rows where it has the “y” class, don’t worry about putting an “n” on this line.